src/ui/components/Assist/Assist.tsx

```diff
@@ -0,0 +1,284 @@
+  console.log("still need to InspectComponent: ", showInspectComponent);
+  console.log("still need to UseFocusMode: ", showUseFocusMode);
+
+  // Hard-coded checklist items
+  const checklistItems = [
+    {
+      label: "Time travel in the console",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video2.gif",
+      completed: showConsoleNavigate,
+    },
+    {
+      label: "Magic print statements",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video3.gif",
+      completed: showBreakpointEdit,
+    },
+    {
+      label: "Add a comment",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video4.gif",
+      completed: showAddComment,
+    },
+    {
+      label: "Jump to code",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video8.gif",
+      completed: showJumpToCode,
+    },
+    {
+      label: "Add a unicorn badge",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video9.gif",
+      completed: showAddUnicornBadge,
+    },
+
+    {
+      label: "Use focus mode",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video18.gif",
+      completed: showUseFocusMode,
+    },
+    {
+      label: "Launch command palette",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video14.gif",
+      completed: showLaunchCommandPalette,
+    },
+    {
+      label: "Record a replay",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video10.gif",
+      completed: showRecordReplay,
+    },
+    {
+      label: "Explore sources",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video11.gif",
+      completed: showExploreSources,
+    },
+    {
+      label: "Search source text",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video12.gif",
+      completed: showSearchSourceText,
+    },
+    {
+      label: "Jump to event",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video15.gif",
+      completed: showJumpToEvent,
+    },
+    {
+      label: "Inspect element",
+      videoUrl: "https://website-git-master-recordreplay.vercel.app/assist/video16.gif",
+      completed: showInspectElement,
+    },
+  ];
+
+  return (
+    <div className={styles.AssistBoxWrapper}>
+      <div className={styles.AssistBoxGradient}>
+        <div className={styles.AssistBox}>
+          <div className="p-0 pt-3">
+            <div className={styles.h1}>Replay Assist</div>
+            <div className={styles.AssistBoxInternal}>
+              <div className={styles.checklist}>
+                {checklistItems.map((item, index) => (
+                  <div
+                    key={index}
+                    className={`flex ${getItemStyle(index)} ${styles.checklistItem}`}
+                    onClick={() => handleClick(index)}
+                  >
```

Comment:
The selected item above is initialized to 1:
```js
const [selectedItem, setSelectedItem] = useState(1);
```

This index-based approach means that the first item in the list is index 0 though. That seems like a bug?

Reply:
Good eye, this was because I wanted to pick that item in the list. Not a bug, but a better solution is "pick the first item that the user hasn't yet completed."